src/draw/lv_draw_vector.c

```diff
@@ -195,6 +195,23 @@ void lv_matrix_multiply(lv_matrix_t * matrix, const lv_matrix_t * m)
     _multiply_matrix(matrix, m);
 }
 
+void lv_matrix_transform_point(const lv_matrix_t * matrix, lv_fpoint_t * point)
```

Comment:
```suggestion
void lv_point_transform_by_matrix(lv_fpoint_t * point, const lv_matrix_t * matrix)
```

Reply:
I think it's weird that there is a separate function called 'lv_point_xxxx' here. Is it  the ‘transform‘ is a method of matrix?

Reply:
I think it classified as a method of matrix， how about that？

Reply:
Okay, let's keep it as it is.